Mathlib/CategoryTheory/Limits/FunctorCategory/Shapes/Kernels.lean

```diff
@@ -0,0 +1,29 @@
+/-
+Copyright (c) 2025 Yaël Dillies, Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Joël Riou
+-/
+import Mathlib.CategoryTheory.Limits.FunctorCategory.Basic
+import Mathlib.CategoryTheory.Limits.Preserves.Shapes.Kernels
+
+/-!
+# (Co)kernels in functor categories
+-/
+
+namespace CategoryTheory.Limits
+universe u
+variable (C : Type*) [Category.{u} C] [HasZeroMorphisms C]
+
+/-- The kernel fork is itself a kernel. -/
```

Comment:
The docstrings here are not very helpful. Could you make it more explicit?

Reply:
I've given it a go